rebasehelper/cli.py

```diff
@@ -200,6 +200,13 @@ def build_parser():
             help=argparse.SUPPRESS
         )
         parser.add_argument(
+            "--pure-html-output",
+            dest="pure_html_output",
+            action="store_true",
+            default=False,
+            help="generate pure html when using html output tool"
```

Comment:
I don't think anybody minds CSS, and if so, one can easily disable (or replace) it oneself. My concern was about javascript, and the usability of the report with it being disabled.